### PR TITLE
releng - try a windows build

### PR DESCRIPTION
**Pull request by kapilt**
see what happens.

### CODE REVIEW

**Comment by kapilt** [comment]
somewhat of a surprise, windows builds worked ootb, https://github.com/cloud-custodian/tfparse/actions/runs/7466017271

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -13,8 +13,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - runner: ubuntu-latest
-            cibw_arch: aarch64
+#          - runner: ubuntu-latest
+#            cibw_arch: aarch64
```
Comment (ajkerrigan) [inline .github/workflows/release.yml:17]: keeping this commented because it's temporary and we want the visual reminder?

**Review by ajkerrigan — APPROVED** [review]
lgtm, yay cibuildwheel doing its thing :sunglasses:

File: .github/workflows/release.yml (hunk 1 above)
Comment (kapilt) [reply .github/workflows/release.yml:17]: yup, those are coming back, just wanted to reduce the cycle time on testing the windows specific portion of the build.

**Comment by kapilt** [comment]
hmm.. doesn't look like windows works in practice, even though it builds, ci failing with
`FAILED tests/test_tfparse.py::test_parse_no_dir - ffi.error: symbol 'free' not found in library 'D:\a\tfparse\tfparse\tfparse.cp312-win_amd64.pyd': error 0x7f`

**Comment by kapilt** [comment]
the function doesn't appear to be exported is the underlying issue, the parse function is, as super hacky thing on windows we're just going to avoid calling free for now and  that means accepting a memory leak :/ the common usage for c7n_left though this is non material as we're effectively exiting after a parse, but its a strong caveat for library usage on win32

**Pull request merged**